### ISSUE TITLE
nothing to lose at a 51% attack

### ISSUE THREAD

**Issue by CJentzsch** [ares-protocol/ares-contracts#2]
Additional to the future rewards described in issue https://github.com/ares-protocol/ares-contracts/issues/1 there is another problem:

A malicious user, who gets a large portion of DAO tokens, can always rob the non-zero number of passive users.
Even if there are no passive users at all, he will get 100% of his money back. Since all active users left the DAO to split into there own DAO, he is the only one left and can still withdraw his funds. So he has nothing to lose doing this. And in the case there is at least one (and most likely there are more) passive user, he can get his/their money.

What's the economic disincentive of attempting this?

**Comment by SilentCicero** [comment]
@CJentzsch yes, we are aware of this, sheer size is what we are hoping prevents this. As well, we have a new decentralized burn incentive scheme that will ensure smaller more naive users have their ether removed from the malicious DAO.

We have a scheme to ensure passive users will always get their money back. Trust me, it's simple, your gonna like it @CJentzsch ;)

**Comment by SilentCicero** [comment]
That solution has not been added yet to this repo, also adding the MiniMe Token as well.

**Comment by CJentzsch** [comment]
Great. Looking forward to it. Especially the passive user protection is crucial.
For the Charity DAO I expect a majority to be passive users. The main reason for the curator in "TheDAO" was to protect passive users. Very curious to see how you will solve that.
Keep me in the loop.

**Comment by SilentCicero** [comment]
Will do, you will like this solve. It's very trust less and crypto-economically incentivized.
